payments/models.py

```diff
@@ -720,7 +721,13 @@ def set_status(self, new_status: OrderStatus, comment: str = None) -> None:
             return
 
         valid_status_changes = {
-            OrderStatus.WAITING: (
+            OrderStatus.DRAFTED: (
+                OrderStatus.OFFERED,
+                OrderStatus.PAID_MANUALLY,
+                OrderStatus.ERROR,
+                OrderStatus.CANCELLED,
```

Comment:
Also, do we need this transition actually? If it's not used in the application flow let's remove it, allowing it later is easy if needed. If the order has not yet been sent to the customer, maybe the correct thing to do would be to remove it completely from the database?

Reply:
Good point! I'll remove this as well.